### PR TITLE
feat(web): token never expiration

### PR DESCRIPTION
**Pull request by robot9706**
Added 'Never' option for user API tokens.

### CODE REVIEW

File: web/crux-ui/src/components/settings/create-token-card.tsx [hunk 1]
```diff
@@ -6,15 +6,15 @@ import { DyoHeading } from '@app/elements/dyo-heading'
...
+import { GeneratedToken, GenerateToken, TokenExpiration } from '@app/models'
 import { API_TOKENS } from '@app/routes'
 import { sendForm } from '@app/utils'
 import { generateTokenSchema } from '@app/validations/token'
 import { useFormik } from 'formik'
 import useTranslation from 'next-translate/useTranslation'
 import { MutableRefObject } from 'react'
 
-const EXPIRATION_VALUES = [30, 60, 90]
+const EXPIRATION_VALUES: TokenExpiration[] = [30, 60, 90, 'never']
```
Comment (polaroi8d) [inline web/crux-ui/src/components/settings/create-token-card.tsx:17]: Not better to use 0 for never, and we can skip check like these `typeof value === 'number'` ?

Comment (robot9706) [reply web/crux-ui/src/components/settings/create-token-card.tsx:17]: I wanted to be explicit about it, but 0 could work too.

Comment (polaroi8d) [reply web/crux-ui/src/components/settings/create-token-card.tsx:17]: Mixing strings with numbers not a good approach IMHO.

File: web/crux/prisma/schema.prisma [hunk 2]
```diff
@@ -28,12 +28,12 @@ model Team {
...
-  userId    String   @db.Uuid
-  name      String   @db.VarChar(70)
-  createdAt DateTime @default(now()) @db.Timestamptz(6)
-  expiresAt DateTime @db.Timestamptz(6)
-  nonce     String   @db.Uuid
+  id        String    @id @default(uuid()) @db.Uuid
+  userId    String    @db.Uuid
+  name      String    @db.VarChar(70)
+  createdAt DateTime  @default(now()) @db.Timestamptz(6)
+  expiresAt DateTime? @db.Timestamptz(6)
```
Comment (polaroi8d) [inline web/crux/prisma/schema.prisma:35]: Nullable?

**Review by polaroi8d — APPROVED** [review]
One minor remark, but LGTM.

**Pull request merged**